brave/brave-core
# Use print preview extraction as fallback when we failed to fetch page content.

Resolves https://github.com/brave/brave-browser/issues/36403

This PR adds a general one time callback so we can use print preview extraction as a final resort whenever page content fetches nothing. (Not limited to image based page but also general sites that we somehow failed to extract)



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Navigate to https://www.vintage-trek.com/Trek-Fisher-Klein-Lemond/2004trek.pdf
2. Open leo panel and summarize the page
3. Response should be relevant
